google/cloud/bigtable/data/_async/_mutate_rows.py

```diff
@@ -104,7 +116,7 @@ def __init__(
         self.timeout_generator = _attempt_timeout_generator(
             attempt_timeout, operation_timeout
         )
-        self.mutations = mutation_entries
+        self.mutations = [_EntryWithProto(m, m._to_pb()) for m in mutation_entries]
```

Comment:
If we make the mutation classes immutable, we can simplify this

Reply:
I think making each individual mutation (ie SetCell, DeleteCell, etc) is a great idea
Howevr I dont think you can make the collection of mutations immutable (ie RowMutationEntry)